### PR TITLE
Add react version and link settings

### LINKED ISSUE #11: Squash react plugin warning
`Warning: React version not specified in eslint-plugin-react settings. See https://github.com/yannickcr/eslint-plugin-react#configuration .`

### PR DESCRIPTION
**Pull request by dpikt**
Resolves #11 

Settings reference: https://github.com/yannickcr/eslint-plugin-react#configuration

### CODE REVIEW

File: react.js [hunk 1]
```diff
@@ -18,6 +18,12 @@ module.exports = {
     ]
   },
   'settings': {
+    'react': {
+      'version': 'detect',
+    },
+    'linkComponents': [
```
Comment (chawes13) [inline react.js]: Can you explain this?

Comment (chawes13) [reply react.js]: Ah, I see the docs you linked to. Do we not need `"Hyperlink"?

Comment (dpikt) [reply react.js]: Yeah, I think that's just another possible link component name. Ala https://www.npmjs.com/package/react-native-hyperlink?

Comment (chawes13) [reply react.js]: ohhh, nice catch! Cool

**Pull request merged**